tests/pytest/test.py

```diff
@@ -308,30 +308,30 @@ def testMgetCommand(env):
     r.assertEqual(len(raw), 5)
     for d in range(0, 5):
         key = 'doc:{}'.format(d)
-        r.assertEqual(json.loads(raw[d]), [docs['basic']], d)
+        r.assertEqual(json.loads(raw[d]), docs['basic'], d)
```

Comment:
I guess this reverts the tests changes?

Reply:
Yes, the previous change ignored the previous behavior. Since the path here starts with a `.`, the old behavior is now kept.